praw/models/user.py

```diff
@@ -53,7 +55,9 @@ def __init__(self, reddit):
 
     def blocked(self):
         """Return a RedditorList of blocked Redditors."""
-        return self._reddit.get(API_PATH["blocked"])
+        data = self._reddit.request("GET", API_PATH["blocked"])
```

Comment:
Can we use `response_data` in this file for consistency?

Reply:
`data` is more common though?